serviceuser.go

```diff
@@ -160,6 +161,23 @@ func (su *ServiceUser) waitUntilReady() error {
 	return nil
 }
 
+// Connect connects to the server at the given "host:port". Either Connect or
+// SetConn must be before calling CStore, etc.
+func (su *ServiceUser) ConnectWithTimeout(serverAddr string, timeout time.Duration) error {
```

Comment:
I think using a context here might be useful later, if we want to graceful terminate the Connect process. I image it should look like

```go
func (su *ServiceUser) ConnectContext(ctx context.Context, serverAddr string) {
	var dialer net.Dialer
	conn, err := dialer.DialContext(ctx, "tcp", serverAddr) 
}

ctx, cancel := context.WithTimeout(time.Minute)
su.ConnectWithTimeout(ctx, ...)
```

Reply:
Fixed at f6f6d07704b9323da4735839ed0f1eee60b0f1f6